scalableminds/webknossos
# Add debugging methods to visualize wireframe of buckets

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Start a tracing and see the buckets in the 3D viewport

I'll remove the actual visualize/unvisualize calls, since they should only be used while debugging.
